chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css

```diff
@@ -1,64 +1,61 @@
 @font-face {
-  font-family: 'fair-chart-icons';
-  src: url('fonts/fair-chart-icons.eot?yr8ymj');
-  src: url('fonts/fair-chart-icons.eot?yr8ymj#iefix') format('embedded-opentype'),
-    url('fonts/fair-chart-icons.ttf?yr8ymj') format('truetype'),
-    url('fonts/fair-chart-icons.woff?yr8ymj') format('woff'),
-    url('fonts/fair-chart-icons.svg?yr8ymj#fair-chart-icons') format('svg');
-  font-weight: normal;
-  font-style: normal;
-  font-display: block;
-}
-
-[class^="icon-"],
-[class*=" icon-"] {
-  /* use !important to prevent issues with browser extensions that change fonts */
-  font-family: 'fair-chart-icons' !important;
-  speak: never;
-  font-style: normal;
-  font-weight: normal;
-  font-variant: normal;
-  text-transform: none;
-  line-height: 1;
-
-  /* Better Font Rendering =========== */
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+    font-family: 'fair-chart-icons';
+    src: url('fonts/fair-chart-icons.eot?yr8ymj');
+    src: url('fonts/fair-chart-icons.eot?yr8ymj#iefix') format('embedded-opentype'),
+        url('fonts/fair-chart-icons.ttf?yr8ymj') format('truetype'), url('fonts/fair-chart-icons.woff?yr8ymj')
+        format('woff'), url('fonts/fair-chart-icons.svg?yr8ymj#fair-chart-icons') format('svg');
+    font-weight: normal;
+    font-style: normal;
+    font-display: block;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+    /* use !important to prevent issues with browser extensions that change fonts */
+    font-family: 'fair-chart-icons' !important;
+    speak: never;
+    font-style: normal;
+    font-weight: normal;
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1;
+    /* Better Font Rendering =========== */
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
 }
 
 .icon-info_icon .path1::before {
-  content: "\49";
-  color: rgb(0, 0, 205);
+    content: "\49";
+    color: rgb(0, 0, 205);
 }
 
 .icon-info_icon .path2::before {
-  content: "\4a";
-  margin-left: -1em;
-  color: rgb(255, 255, 255);
+    content: "\4a";
+    margin-left: -1em;
+    color: rgb(255, 255, 255);
 }
 
 .icon-warn_icon .path1::before {
-  content: "\57";
-  color: rgb(255, 215, 0);
+    content: "\57";
+    color: rgb(255, 215, 0);
 }
 
 .icon-warn_icon .path2::before {
-  content: "\58";
-  margin-left: -1.1376953125em;
-  color: rgb(0, 0, 0);
+    content: "\58";
+    margin-left: -1.1376953125em;
+    color: rgb(0, 0, 0);
 }
 
 .icon-error_icon .path1::before {
-  content: "\45";
-  color: rgb(237, 28, 36);
+    content: "\45";
+    color: rgb(237, 28, 36);
 }
 
 .icon-error_icon .path2::before {
-  content: "\46";
-  margin-left: -1em;
-  color: rgb(255, 255, 255);
+    content: "\46";
+    margin-left: -1em;
+    color: rgb(255, 255, 255);
 }
 
 .text {
-  -fx-font-family: "fair-chart-icons";
-}
+    -fx-font-family: "fair-chart-icons";
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780860)